traefik/templates/_podtemplate.tpl

```diff
@@ -942,9 +957,24 @@
           {{- toYaml .Values.deployment.additionalVolumes | nindent 8 }}
         {{- end }}
         {{- range $localPluginName, $localPlugin := .Values.experimental.localPlugins }}
+        {{- $pluginType := include "traefik.getLocalPluginType" (dict "plugin" $localPlugin "pluginName" $localPluginName) }}
+        {{- if ne $pluginType "localPath" }}
```

Comment:
And here we would probably be able to just do if type === x, then (print body), if type === y (print body), ... else "error unknown plugin type.